src/helpers/AuthContext.tsx

```diff
@@ -44,20 +45,21 @@ function userReducer(state: UserSessionData, action: Action): UserSessionData {
     }
 
     case 'LOGIN':
-      const newState = {
+      const newState: UserSessionData = {
         ...state,
         token: action.payload!.token,
         orgMembership: action.payload!.orgMembership,
         appId: action.payload!.appId,
         firstName: action.payload!.firstName,
         lastName: action.payload!.lastName,
-        userName: action.payload!.username,
+        username: action.payload!.username,
         dataGroups: action.payload!.dataGroups,
```

Comment:
The field name had a small typo, identified by specifying the type for `newState`.